libraries/networking/src/DomainHandler.h

```diff
@@ -199,7 +199,8 @@ class DomainHandler : public QObject {
         Unknown,
         ProtocolMismatch,
         LoginError,
-        NotAuthorized,
+        NotAuthorizedMetaverse,
+        NotAuthorizedDomain,
```

Comment:
The new reason should be appended to the end of the list in order to maintain compatibility with previous releases on the same prototol and any scripts using the value. I'll move its location in another PR.